examples/kyb-app/src/components/organisms/UIRenderer/elements/JSONForm/helpers/findDefinitionByName.ts

```diff
@@ -0,0 +1,42 @@
+import { UIElement } from '@app/domains/collection-flow';
+import { AnyObject } from '@ballerine/ui';
+
+export const findDefinitionByName = (
+  name: string,
+  elements: UIElement<AnyObject>[],
+): UIElement<AnyObject> => {
+  for (const element of elements) {
+    if (element.name === name) {
+      return element;
+    }
+
+    if (element.elements) {
+      const foundInChildren = findDefinitionByName(name, element.elements);
+      if (foundInChildren) {
+        return foundInChildren;
+      }
+    }
+  }
+
+  return undefined;
+};
+
+export const findDefinitionByDestinationPath = (
+  destination: string,
+  elements: UIElement<AnyObject>[],
+): UIElement<AnyObject> | undefined => {
+  for (const element of elements) {
+    if (element.valueDestination === destination) {
+      return element;
+    }
+
+    if (element.elements) {
+      const foundInChildren = findDefinitionByDestinationPath(destination, element.elements);
+      if (foundInChildren) {
+        return foundInChildren;
+      }
```

Comment:
If `foundInChildren` is `undefined` then that is what returned anyways.
```suggestion
        return foundInChildren;
```